nowcasting_dataset/data_sources/gsp/gsp_data_source.py

```diff
@@ -41,11 +41,9 @@ class GSPDataSource(ImageDataSource):
     # zarr_path of where the gsp data is stored
     zarr_path: Union[str, Path]
     # start datetime, this can be None
-    # TODO: Issue #425: Use config to set start_dt and end_dt.
-    start_dt: Optional[datetime] = pd.Timestamp("2020-01-01")
+    start_datetime: Optional[datetime] = pd.Timestamp("2020-01-01")
```

Comment:
I'd be very tempted to remove these defaults.  Defaults like this can, all too easily, mean that the code silently does the wrong thing!  Or to set the default to `None` (if `None` means "use the extent of the dataset")

Reply:
Yea I can put them as None